pkg/common/common.go

```diff
@@ -3,23 +3,29 @@ package common
 import "fmt"
 
 const (
-	Velero             = "velero"
-	Restic             = "restic"
-	VeleroNamespace    = "oadp-operator"
-	OADPOperator       = "oadp-operator"
-	OADPOperatorVelero = "oadp-operator-velero"
+	Velero                       = "velero"
+	Restic                       = "restic"
+	VeleroNamespace              = "oadp-operator"
+	OADPOperator                 = "oadp-operator"
+	OADPOperatorVelero           = "oadp-operator-velero"
+	DataMover                    = "data-mover"
+	DataMoverController          = "data-mover-controller"
+	DataMoverControllerContainer = "data-mover-controller-container"
+	OADPOperatorServiceAccount   = "openshift-adp-controller-manager"
 )
 
 // Images
 const (
-	VeleroImage          = "quay.io/konveyor/velero:latest"
-	OpenshiftPluginImage = "quay.io/konveyor/openshift-velero-plugin:latest"
-	AWSPluginImage       = "quay.io/konveyor/velero-plugin-for-aws:latest"
-	AzurePluginImage     = "quay.io/konveyor/velero-plugin-for-microsoft-azure:latest"
-	GCPPluginImage       = "quay.io/konveyor/velero-plugin-for-gcp:latest"
-	CSIPluginImage       = "quay.io/konveyor/velero-plugin-for-csi:latest"
-	RegistryImage        = "quay.io/konveyor/registry:latest"
-	KubeVirtPluginImage  = "quay.io/konveyor/kubevirt-velero-plugin:v0.2.0"
+	VeleroImage             = "quay.io/konveyor/velero:latest"
+	OpenshiftPluginImage    = "quay.io/konveyor/openshift-velero-plugin:latest"
+	AWSPluginImage          = "quay.io/konveyor/velero-plugin-for-aws:latest"
+	AzurePluginImage        = "quay.io/konveyor/velero-plugin-for-microsoft-azure:latest"
+	GCPPluginImage          = "quay.io/konveyor/velero-plugin-for-gcp:latest"
+	CSIPluginImage          = "quay.io/konveyor/velero-plugin-for-csi:latest"
+	CSIDataMoverPluginImage = "quay.io/konveyor/velero-plugin-for-csi:data-mover"
+	DataMoverImage          = "quay.io/konveyor/volume-snapshot-mover:latest"
```

Comment:
Can you comment the differences between these two?

Reply:
Added comments